mail-filter/rspamd/rspamd-9999.ebuild

```diff
@@ -53,6 +54,8 @@ src_configure() {
 		-DENABLE_LUAJIT=$(usex jit ON OFF)
 		-DENABLE_FANN=$(usex fann ON OFF)
 		-DENABLE_GD=$(usex gd ON OFF)
+		-DENABLE_PCRE2=$(usex pcre2 ON OFF)
+		-DENABLE_JEMALLOC=$(usex jemalloc ON OFF)
```

Comment:
in theory you can drop all these ON OFF thingies, as cmake interprets `yes` and `no` too (https://cmake.org/cmake/help/v3.0/command/if.html)